VERSION_INFO

```diff
@@ -1 +1 @@
-0.2.0rc5
\ No newline at end of file
+0.2.0rc6
```

Comment:
Is this newline needed? I don't remember.

Reply:
the new line needs to be removed